src/app/pages/LandingPage.svelte

```diff
@@ -7,7 +7,9 @@
   let downloadURL = "";
 
   worker.onPackageCompileSuccess(
-    Comlink.proxy((kmp: ArrayBuffer) => {
+    Comlink.proxy(async () => {
+      const KMPPackage = await worker.getKMPPackage();
+      const kmp = KMPPackage.package;
```

Comment:
See suggestions below

```suggestion
      const kmp = await worker.getKMPPackage();
```